### DISCUSSION TITLE
init-frz-model from a frozen_model.pd

### DISCUSSION THREAD

**Discussion by wei0py** [deepmodeling/deepmd-kit#1722] [Q&A]
When I use dp train --init_frz_model frozen_model.pb input.json, I came across the problem:
RuntimeError: The input frozen model: frozen_model.pb has no 'model_type' information, which is not supported by the 'dp train init-frz-model' interface.

What's more, if I use the convert-from to convert the frozen_model.pb from 1.2 and then use the --init-frz-model, the new error tells me:
ValueError: Node 'model_attr/model_version' is not unique

Is there any way to solve it? Thanks!

**Comment by njzjz** [comment]
Currently, `model_type` is not supported to add by converting.
